zmon_aws_agent/main.py

```diff
@@ -189,6 +197,8 @@ def main():
         certificates + sqs)
     current_entities.append(aws_limits)
     current_entities.append(ia_entity)
+    for entity in current_entities:
+        entity.update('extras', entity_extras)
```

Comment:
fields should be first level, so ``entity.update(entity_extras)`` should be sufficient

Reply:
that means that someone can overwrite discovered keys with these values, like `EXTRA_ENTITY_FIELDS="dns_traffic=true"`

Reply:
yes they can, and it is their problem then 😉

Reply:
it is important also because ZMON doesn't support entity filters for nested fields.